build-caching-maven-samples/spotbugs-project/pom.xml

```diff
@@ -133,6 +135,10 @@
                             <name>xmlOutputFile</name>
                             <path>${project.build.directory}/spotbugs.xml</path>
                           </file>
+                          <file>
+                            <name>htmlOutputFile</name>
+                            <path>${project.build.directory}/spotbugs.html</path>
```

Comment:
I verified this was the proper location for the file.

https://github.com/spotbugs/spotbugs-maven-plugin/pull/161/files#diff-a8270ec7df0fe9dcf1a1e8120b333c540187439faf5963d157f3a3e846a851e6R1091